src/components/GraphFilter/LayoutDropdown.tsx

```diff
@@ -20,33 +21,49 @@ export class LayoutDropdown extends React.Component<LayoutDropdownProps, LayoutD
     ['dagre', 'Dagre'],
     ['klay', 'Klay']
   ];
+  static InitialLabel = 'Select Graph Type';
```

Comment:
This is more of a question on style: Do we want to use ALL_CAPS for static types or ??

Reply:
I am working in a JIRA to create a style code guide, in some places there is car with camelcase and orhers with uppercase for example. We need to have the code more aligned

Reply:
Yeah I think in some cases I also wrote upper camel case for static fields. We must define our standard. I have no opinion about what is better.